### PR TITLE
Update Rehydration workflow (new features, redesign)

### PR DESCRIPTION
**Pull request by pcarioufr**
### Preview link

https://docs-staging.datadoghq.com/pcarioufr/rehydration_estimate_notify/logs/archives/rehydrating/?tab=awss3#add-new-historical-views

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/logs/archives/rehydrating.md [hunk 1]
```diff
@@ -20,21 +20,23 @@ Index exclusion filters do not apply to historical views, so there is no need to
...
-3. **Input the query**. The query syntax is the same as that of the [log explorer search][4], but is limited to log attributes, [reserved attributes][5], and free text search on the message.
+3. **Select the archive** from which you wish to rehydrate log events. Only archives that are [configured to use role delegation](#permissions) are available for rehydrating.
 
-4. **Name your historical view**. Names must begin with a lowercase letter and can only contain lowercase letters, numbers, and the `-` character.
+4. (Optional) **estimate scan size** and get the total amount of compressed data that is contained in your archive for the selected timeframe.
 
-5. (Optional) **Add a description** to give your team context about the purpose of the historical view.
+5. **Name your historical view**. Names must begin with a lowercase letter and can only contain lowercase letters, numbers, and the `-` character.
 
 6. Define the maximum number of logs that should be rehydrated in this historical view, **from 1 million to 1 billion**
```
Comment (sarina-dd) [inline content/en/logs/archives/rehydrating.md:33]:
```suggestion
6. Define the maximum number of logs that should be rehydrated in this historical view, **from 1 million to 1 billion**.
```

File: content/en/logs/archives/rehydrating.md [hunk 2]
```diff
@@ -20,21 +20,23 @@ Index exclusion filters do not apply to historical views, so there is no need to
...
 
 7. Define the retention period of the rehydrated logs (available retentions are based on your contract, default is 15 days)
```
Comment (sarina-dd) [inline content/en/logs/archives/rehydrating.md:35]:
```suggestion
7. Define the retention period of the rehydrated logs (available retentions are based on your contract, default is 15 days).
```

File: content/en/logs/archives/rehydrating.md [hunk 3]
```diff
@@ -20,21 +20,23 @@ Index exclusion filters do not apply to historical views, so there is no need to
 
 ### Add new historical views
 
-1. **Select the archive** from which you wish to rehydrate log events. Only archives that are [configured to use role delegation](#permissions) are available for rehydrating.
+1. **Choose the time period** for which you wish to rehydrate log events.
 
-2. **Choose the time period** for which you wish to rehydrate log events.
+2. **Input the query**. The query syntax is the same as that of the [log explorer search][4], but is limited to log attributes, [reserved attributes][5], and free text search on the message.
```
Comment (sarina-dd) [inline content/en/logs/archives/rehydrating.md:25]: The reserved attributes link takes you to the logs overview page, is this the correct link?

File: content/en/logs/archives/rehydrating.md [hunk 4]
```diff
@@ -20,21 +20,23 @@ Index exclusion filters do not apply to historical views, so there is no need to
...
+5. **Name your historical view**. Names must begin with a lowercase letter and can only contain lowercase letters, numbers, and the `-` character.
 
 6. Define the maximum number of logs that should be rehydrated in this historical view, **from 1 million to 1 billion**
 
 7. Define the retention period of the rehydrated logs (available retentions are based on your contract, default is 15 days)
 
-{{< img src="logs/archives/log_archives_rehydrate_reload.png" alt="Reload from Archive"  style="width:75%;">}}
+8. (Optional) **Notify** trigger notifications on rehydration completion through [integrations][7] with the @handle syntax.
+
+{{< img src="logs/archives/log_rehydration_setup.png" alt="Reload from Archive"  style="width:75%;">}}
```
Comment (sarina-dd) [inline content/en/logs/archives/rehydrating.md:39]: Image is broken here

File: content/en/logs/archives/rehydrating.md (hunk 3 above)
Comment (pcarioufr) [reply content/en/logs/archives/rehydrating.md:25]: nice catch! no it's not. Some link lost in translation. I fixed it.

**Pull request merged**